src/api-v1.js

```diff
@@ -90,10 +91,12 @@ api.declare({
     let result;
 
     do {
+      let startTime = new Date();
```

Comment:
I'm wrong... timedHandler is something very very different...

But we could still add a time(prefix, handlerOrPromise) method to monitor... Making it easy to measure time of a function, async function or promise.